### ISSUE TITLE
playback defaults to minor triad when interpreting unrecognized chord modifiers starting with "maj"

### ISSUE THREAD

**Issue by chrisfargen** [paulrosen/abcjs#368]
Here's some example input:

```
%%text Expected playback
"Gmaj7(9)" Z |
%%text Unexpected playback
"Gmaj7add9" Z |
```

Here's a possible fix for most use cases. In file "abc_midi_flattener.js", function "chordNotes()":

```
	function chordNotes(bass, modifier) {
		var intervals = chordIntervals[modifier];
		if (!intervals) {
			/* common symbols for major */
			if (modifier.slice(0,2).toLowerCase() === 'ma' || modifier.charAt(0) === 'M')
				intervals = chordIntervals.M;
			/* common symbols for minor */
			else if (modifier.charAt(0) === 'm' || modifier.charAt(0) === '-')
				intervals = chordIntervals.m;
			else
				intervals = chordIntervals.M;
		}
		...
```

**Comment by paulrosen** [comment]
I'll test that out soon. In the meantime, you can submit it as a pull request so you get the credit for the fix. If you don't want to, I'll just add it.

**Comment by chrisfargen** [comment]
It's all you. Thanks.

**Comment by paulrosen** [comment]
Will be fixed in 6.0.0-beta.10
